R/github_handling.R

```diff
@@ -0,0 +1,290 @@
+#' Retrieve pages url for a repo
```

Comment:
For now this is a direct copy over from cow. But I'll be doing some reworking of this #157